### ISSUE TITLE
对比学习预训练模型

### ISSUE THREAD

**Issue by jqwangai** [iantangc/ContrastiveLearningHAR#1]
作者你好，请问一下你的对比学习模型的初始参数是在哪个数据集上训练的。如 SimCLR中的预训练参数是使用ResNet在ImageNet上训练的。

**Comment by iantangc** [comment]
Hello! The models were initialised with random weights, and pre-trained directly using the MotionSense dataset on the contrastive learning task, after it has been split into training and testing sets. After contrastive pre-training, the encoding head was removed, and embeddings were extracted from a intermediate layer. Depending on the evaluation scheme, a linear layer or multiple dense layers were used for obtaining the softmax scores for final classification. You may refer to the demo notebook (https://github.com/iantangc/ContrastiveLearningHAR/blob/main/SimCLR_MotionSense.ipynb) for the training settings outlined in our paper.
